src/NodaTime.Web/.gitignore

```diff
@@ -0,0 +1,2 @@
+wwwroot/js/*.min.js
+wwwroot/css/*.min.css
```

Comment:
Let's list `site.min.js` and `site.min.css` explicitly; these two currently also match the foundation minified JS/CSS, which we don't want to ignore.